packages/core/src/components/icon/readme.md

```diff
@@ -34,7 +34,6 @@
  - [dso-pagination](../pagination)
  - [dso-table](../table)
  - [dso-tree-view](../tree-view)
- - [dso-viewer-grid](../viewer-grid)
```

Comment:
@LukeVerlaan Dit is eigenlijk heel raar.. Want de dependency is er nog steeds, maar nu via een Functional Component. Ik heb dit in de Stencil community aangekaart. Dit zou de bevindingen in #2193 verklaren.